pkg/authorization/registry/util/subject.go

```diff
@@ -0,0 +1,37 @@
+package util
+
+import (
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/apiserver/pkg/authentication/serviceaccount"
+	"k8s.io/kubernetes/pkg/apis/rbac"
+)
+
+func GetAllowedSubjects(namespace string, subjects []rbac.Subject) (sets.String, sets.String) {
```

Comment:
`func ExpandSubjects(namespace string, subjects []rbac.Subject) ([]string, []string)` and move to `pkg/authorization/util/subject.go`

Technically we should have fuzzing tests for round tripping between this and `BuildRBACSubjects`.